production-deployment-using-tiup.md

```diff
@@ -2,14 +2,14 @@
 title: Deploy a TiDB Cluster Using TiUP
 summary: Learn how to easily deploy a TiDB cluster using TiUP.
 category: how-to
-aliases: ['/docs/dev/how-to/deploy/orchestrated/tiup/']
+aliases: ['/docs/dev/how-to/deploy/orchestrated/tiup/','/docs-cn/dev/tiflash/deploy-tiflash/']
```

Comment:
```suggestion
aliases: ['/docs/dev/how-to/deploy/orchestrated/tiup/']
```
deploy-tiflash has not been deleted yet, so we should add this alias later when aligning https://github.com/pingcap/docs-cn/pull/3532/